roles.json

```
@@ -587,7 +587,8 @@
                 "role": "reproject",
                 "people": [
                     "Stuart Mumford",
-                    "Thomas Robitaille"
+                    "Thomas Robitaille",
+                    "Sam van Kooten"
```

```suggestion
                    "Sam Van Kooten"
```
My family's far enough removed from the Netherlands that we capitalize the V